Jenkinsfile

```diff
@@ -1,49 +1,90 @@
-@Library('xmos_jenkins_shared_library@v0.32.0') _
+@Library('xmos_jenkins_shared_library@develop') _
```

Comment:
I guess you'll want to update this when you can.

Reply:
Yes, I'll make a separate PR changing this when jenkins_shared_library is released.